### PR TITLE
S dh 462 Base Package

### PR DESCRIPTION
**Pull request by byarsawich**

### CODE REVIEW

File: src/classes/Contacts.cls-meta.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
-  <apiVersion>41.0</apiVersion>
-  <status>Active</status>
+	<apiVersion>41.0</apiVersion>
```
Comment (caraelizc) [inline src/classes/Contacts.cls-meta.xml]: Should this be 43?

File: src/classes/Test_AccountContactRelationService.cls [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+		);
+
+		Contact c3 = new Contact(
+				LastName = 'Prime'
+		);
+
+
+		Database.insert(c1, dml);
+		Database.insert(c2, dml);
+		Database.insert(c3, dml);
```
Comment (caraelizc) [inline src/classes/Test_AccountContactRelationService.cls]: You could add these records to a list and run one database insert.

File: src/classes/Test_AccountService.cls [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+		);
+
+		Test.startTest();
+
+		Database.DMLOptions dml = new Database.DMLOptions();
+		dml.DuplicateRuleHeader.AllowSave = true;
+		Database.SaveResult sr = Database.insert(a1, dml);
+		Database.SaveResult sr2 = Database.insert(a2, dml);
+		Database.SaveResult sr3 = Database.insert(a3, dml);
+		Database.SaveResult sr4 = Database.insert(a4, dml);
```
Comment (caraelizc) [inline src/classes/Test_AccountService.cls]: Consolidate into one insert

File: src/classes/ContactService.cls [hunk 4]
```diff
@@ -1,41 +1,57 @@
 /*
- * Created by byarsawi on 7/17/2018.
+ * Created by byarsawi on 9/6/2018.
```
[the remark this answers is not in the document]
Comment (byarsawich) [reply src/classes/ContactService.cls]: There's a long story, but the TL;DR is I may have accidentally deleted this file and had to recreate it.

[the remark this answers is not in the document]
Comment (byarsawich) [reply src/classes/ContactService.cls]: I changed it back to 7/17

**Comment by caraelizc** [comment]
APPROVED

**Pull request merged**